lms/djangoapps/instructor/tests/views/test_instructor_dashboard.py

```diff
@@ -231,6 +231,43 @@ def test_staff_can_see_writable_gradebook(self):
         self.assertIn(expected_gradebook_url, response.content)
         self.assertIn('View Gradebook', response.content)
 
+    GRADEBOOK_LEARNER_COUNT_MESSAGE = (
+        'Note: This feature is available only to courses with a small number ' +
+        'of enrolled learners.'
+    )
+
+    def test_gradebook_learner_count_message(self):
+        """
+        Test that, when the writable gradebook featue is NOT enabled, there IS
+        a message that the feature is only available for courses with small
+        numbers of learners.
+        """
+        response = self.client.get(self.url)
+        self.assertIn(
+            TestInstructorDashboard.GRADEBOOK_LEARNER_COUNT_MESSAGE,
```

Comment:
Nit: you should be able to refer to class-scoped globals like this using `self`.

Reply:
good point; thanks!